test/corpus/constants.txt

```diff
@@ -3,7 +3,7 @@ Constants
 ================================================================================
 
 const a = "hello"
-const a = "hello\nworld\!"
```

Comment:
`\!` is not a valid escape sequence, and the Gleam compiler does not accept it. As of this update, the tree-sitter grammar no longer accepts it either! Strange this wasn't caught until now.